plugins/modules/proxmox_node_info.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright John Berninger (@jberning) <john.berninger at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: proxmox_node_info
+short_description: Retrieve information about one or more Proxmox VE nodes
+description:
+  - Retrieve information about one or more Proxmox VE nodes.
+author: John Berninger (@jberning)
+extends_documentation_fragment:
+  - community.general.proxmox.documentation
+  - community.general.attributes
+  - community.general.attributes.info_module
+'''
+
+
+EXAMPLES = '''
+- name: List existing nodes
+  community.general.proxmox_node_info:
+    api_host: proxmox1
+    api_user: root@pam
+    api_password: "{{ password | default(omit) }}"
+    api_token_id: "{{ token_id | default(omit) }}"
+    api_token_secret: "{{ token_secret | default(omit) }}"
+  register: proxmox_nodes
+'''
+
+
+RETURN = '''
+proxmox_nodes:
+    description: List of Proxmox VE nodes.
+    returned: always, but can be empty
+    type: list
+    elements: dict
+    contains:
+      cpu:
+        description: Current CPU usage in fractional shares of this host's total available CPU
```

Comment:
All descriptions should consist out of one or multiple paragraphs with complete sentences.
```suggestion
        description: Current CPU usage in fractional shares of this host's total available CPU.
```
Same below.